### ISSUE TITLE
Receiver won't deactivate

### ISSUE THREAD

**Issue by karai17** [WirelessRedstoneGroup/WirelessRedstone#181]
I had an active channel set up and when I slapped the receiver, it gave me the correct error saying I could not remove it. However, when I switched the channel off, the receiver continued to act as if it we active. I removed everything on that channel except for the bugged receiver and it tried to remove the channel but I got an error saying channel does not exist. The receiver (clock) continues to blink.

**Comment by karai17** [comment]
Right-clicking the bugged sign re-activated the channel. I was able to create a transmitter and switch it on then off to deactivate the receiver.

**Comment by Bart-0110** [comment]
So you had a clock receiver and try to break it while the torch was visible or when the sign was visible?

**Comment by karai17** [comment]
I cannot recall which state the clock was in, but the channel was active.
